MVP/Team #5/app.py

```diff
@@ -0,0 +1,59 @@
+import streamlit as st
+import pandas as pd
+from model_logic import load_data, train_model, calculate_cost, plot_training_steps
+import plotly.graph_objects as go
+
+st.set_page_config(page_title="Regresja liniowa", layout="wide")
+
+# Nagłówek
+st.markdown("<h1 style='text-align: center; color: #006699;'>Regresja liniowa</h1>", unsafe_allow_html=True)
+
+# Wczytywanie pliku
+uploaded_file = st.file_uploader("Prześlij plik CSV", type="csv", key="fileUploader")
+
+if uploaded_file:
+    data, X, y = load_data(uploaded_file)
+
+    # Podgląd danych
+    with st.container():
+        st.markdown("<h3 style='color: #003366;'>Podgląd danych:</h3>", unsafe_allow_html=True)
+        st.dataframe(pd.DataFrame({
+            'Doświadczenie': X.flatten(),
+            'Wynagrodzenie': y.astype(str)
+        }), height=150)
+
+    st.markdown("---")
+
+    # Wybór funkcji kosztu
+    cost_function = st.selectbox("Wybierz funkcję kosztu", ["Błąd średniokwadratowy (MSE)", "Błąd średniobezwzględny (MAE)", "R2 Score"])
+
+    # Ustawienia modelu
+    col1, col2, col3 = st.columns([1, 1, 1])
+    with col1:
+        regularization_type = st.selectbox("Wybierz typ regularyzacji", ["None", "Lasso", "Ridge"])
+    with col2:
+        alpha = st.slider("Wybierz parametr regularyzacji (alpha)", 0.01, 10.0, 1.0)
```

Comment:
Please explain the meaning of these constants: 0.01, 10.0, 1.0 - see rule Use Names That Speak for Themselves (MN 01)